vowpalwabbit/parse_args.cc

```diff
@@ -1532,14 +1536,14 @@ void parse_modules(options_i& options, VW::workspace& all, bool interactions_set
   options.add_and_parse(rand_options);
   all.get_random_state()->set_random_state(all.random_seed);
 
-  parse_update_options(options, all);
-
   parse_feature_tweaks(options, all, interactions_settings_duplicated, dictionary_namespaces);  // feature tweaks
 
   parse_example_tweaks(options, all);  // example manipulation
 
   parse_output_model(options, all);
 
+  parse_update_options(options, all);
```

Comment:
Is this reorder still needed?

Reply:
Yes because `parse_output_model(options, all);` determings `all.save_resume` which is used in `parse_update_options(options, all);`

Reply:
Got it thanks!